analog-in/cpp/analog-in.cpp

```diff
@@ -35,8 +35,23 @@
 using namespace std;
 using namespace mraa;
 
+// check if running as root
+void CheckRoot(void)
+{
+	int euid = geteuid();
+	if (euid) {
+		cerr << "This project uses Mraa I/O operations, but you're not running as 'root'.\n"
+				"The IO operations below might fail.\n"
```

Comment:
Maybe github online is making it look like this but is this 4 space/1 tab indented? We should be consistent with whatever the rest of the sample uses.

Reply:
Yes, this is 1 tab